sinara.py

```diff
@@ -62,7 +62,8 @@ class Sinara(_SinaraTuple):
         "Booster",
         "Booster_Channel",
         "DIO_MCX",
-        "Kasli_soc"
+        "Kasli_soc",
+        "Mirny_Almazny"
```

Comment:
It's just "Almazny".
And it's not a PLL. Just "4x 5-12 GHz mezzanine".

Reply:
Well, `Mirny_Almazny` was intentional.

We do not have separate EEPROM for Almazny. Functionality of Mirny is not altered by using Almazny. So identified device is still Mirny.

For the description, I would rather change Mirny to "4x 53-4000MHz PLL" and Mirny_Almazny to "53-12000 MHz PLL"

Reply:
It wasn't clear that this is about the eeprom in mirny, not on almazny. I thought there was also one on almazny. Why do we need a different device id? Does the variant not suffice?

Reply:
There is a downside of my `Mirny_Almazny` approach which is the question of what would be the version of such device? Version of Almazny, version of Mirny or what?

It would be better to have a dedicated EEPROM on Almazny. Due to an issue with invalid connection a new revision will probably be needed, so we could include mezzanine dedicated EEPROM (https://github.com/sinara-hw/Almazny/issues/7#issuecomment-1322378221).

Reply:
Variant may suffice, although it still does not address the version issue.

Reply:
Version is just what the version mirny is.
You can also use a byte of the 64 that are there for board data to encode almazny presence and almazny version, Gateware version etc.

Reply:
OK, so we're making it a variant: `"Mirny": [ "Mirny", "Mirny_Almazny"]`

Board data:
```
"name",          # 10s, name of the mezzanine, human redable
"board",         # H, mezzanine ID
"major",         # B, board major revision
"minor",         # B, board minor revision
"variant",       # B, board variant
"vendor",        # B, vendor id
"vendor_data",   # 8s, vendor reserved: serial number/data
```

If you agree I'll make changes to the PR.

Reply:
Actually, I meant that those board data I listed would be packed into `board_data` field. Using both variant and board data would solve the problem IMO (Almazny variant -> we know how to interpret board_data, and board_data identifies Almazny device).

Reply:
that would reserve a lot. I'd rather say that since almazny doesn't yet have an eeprom we should only put the absolutely minimal data into mirny instead and not restrict mirny unnecessarily.

Reply:
For urukul we use it extensively.
Variant plus 2 B board data sounds fine. Let's document that in the code